io.github.cgueret.Scriptorium.json

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "io.github.cgueret.Scriptorium",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "48",
+    "sdk": "org.gnome.Sdk",
+    "command": "scriptorium",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "scriptorium",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "dir",
+                    "path": "./"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "make-args": [
+                "NO_TCLTK=1",
+                "INSTALL_SYMLINKS=1"
+            ],
+            "make-install-args": [
+                "NO_TCLTK=1",
+                "INSTALL_SYMLINKS=1"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.45.2.tar.xz",
+                    "sha256": "51bfe87eb1c02fed1484051875365eeab229831d30d0cec5d89a14f9e40e9adb"
+                }
+            ]
+        },
+	"python3-ebooklib.json",
+	"python3-beautifulsoup4.json",
+	"python3-pyyaml.json",
+	"python3-GitPython.json",
```

Comment:
neither of these 4 exist.

Reply:
Indeed they are in the repo but not in that set of files. If I had them is that ok or is the better approach to copy & paste their content into the manifest?

Reply:
Added them to the repo